features/event-simulator/org.wso2.carbon.event.simulator.server.feature/src/main/resources/conf/rdbmsConfiguration.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2014-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ WSO2 Inc. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+-->
+<!--
+
+ Please consider following queries.
+
+ - "INSERT INTO alerts_history ( id,information,latitude,longitude,state,timeStamp) VALUES (?,?,?,?,?,?)"
+ - "CREATE TABLE alerts_history (ID  varchar2(255),INFORMATION  varchar2(255),LATITUDE  BINARY_DOUBLE,LONGITUDE  BINARY_DOUBLE,STATE  varchar2(255),TIMESTAMP  varchar2(255))"
+ - "UPDATE alerts_history SET information = ?  , latitude = ?  , longitude = ?  , state = ?  , timeStamp = ?  WHERE id = ? "
+
+
+ Following attributes corresponds to following values
+
+ $TABLE_NAME - alerts_history
+ $COLUMN_TYPES - ID  varchar2(255),INFORMATION  varchar2(255),LATITUDE  BINARY_DOUBLE,LONGITUDE  BINARY_DOUBLE,STATE  varchar2(255),TIMESTAMP  varchar2(255)
+ $COLUMNS - id,information,latitude,longitude,state,timeStamp
+ $VALUES - ?,?,?,?,?,?
+ $COLOUMN_VALUES - information = ?  , latitude = ?  , longitude = ?  , state = ?  , timeStamp = ?
+ $CONDITION - id = ?
+
+-->
+
+<adaptorConfiguration>
```

Comment:
I was not referring to the file name, I was referring to the tag <adaptorConfiguration>
It has to be <rdbmsConfiguration> use the rdbms-configuration.xml for the file name